site/docs/glossary/types.md

```diff
@@ -13,3 +17,23 @@ SourceHash uniquely identifies the source of a deposit transaction and serves as
 ## SourceHashDomain
 
 SourceHashDomain specifies whether the deposit transaction is from a deposit on L1 or is a system transaction. You can see the type [here](https://github.com/base-org/op-viem/blob/f6595e8a0373f79e17b3c2c89f486091833e4d17/src/types/depositTransaction.ts#L5-L8).
+
+## TransactionDepositedEvent
+
+An [event](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L1/OptimismPortal.sol#L73C1-L73C1) emitted from a [depositTransaction](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L1/OptimismPortal.sol#L377) call to the [OptimismPortal](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L1/OptimismPortal.sol) contract, which is used to create a [DepositTransaction](#deposittransaction) on L2.
+
+## WithdrawTransaction
```

Comment:
So WithdrawalTransaction is not a transaction type in the way that DepositTransaction is (which exists as a type in Op-Geth). It's really just the `MessagePassedEvent` which gives info that gets passed around

Reply:
Or I think we could just update this all to just say "An L2 -> L1 transaction." We could possible keep your info about the various steps, or maybe it belongs in more of a"flows" docs. I have tried to link to some docs OP has where it makes sense.

Reply:
@Sabnock01 lmk if you're going to update this one or if someone else should :)